lib/ansible/playbook/role/__init__.py

```diff
@@ -460,17 +466,29 @@ def get_dep_chain(self):
             dep_chain.append(parent)
         return dep_chain
 
+    def get_cached_deps(self):
```

Comment:
This could be a `@property`.

Reply:
This is acting upon `global_role_dependency_cache` which is a module-wide variable, not a class-member. I could change the call into a property (or their module-equivalent, as suggested [in this thread](https://stackoverflow.com/a/57263518/199513)), but I'm not sure that'll add readability.

Reply:
It's just cleaner to have a `.cached_deps` in place of `.get_cached_deps()`, I think, since the latter implies an imperative action and kinda suggests that maybe and argument would be expected, but there's none.